src/lib/fragments/AgGrid.react.js

```diff
@@ -386,6 +388,25 @@ export default class DashAgGrid extends Component {
                     return c;
                 });
             }
+            if (OBJ_MAYBE_FUNCTION_OR_MAP_MAYBE_FUNCTIONS[target]) {
+                if ('function' in value) {
+                    if (typeof value.function === 'string') {
+                        return this.convertMaybeFunctionNoParams(value);
+                    }
+                }
+                return mapObjIndexed((v) => {
```

Comment:
```suggestion
                return map((v) => {
```
minor point, but no need to use the `indexed` form if you're only using the values, not the keys